App/CMakeLists.txt

```diff
@@ -18,10 +18,10 @@
 # ***** END LICENSE BLOCK *****
 
 set(Natron_SOURCES NatronApp_main.cpp)
-if(WINDOWS)
-    list(APPEND Natron_SOURCES ../Natron.rc)
-endif()
 add_executable(Natron ${Natron_SOURCES})
+if(WIN32)
```

Comment:
I can't review the WIN32 stuff, @acolwell can you give it a look?

Reply:
I agree. The CMake stuff should be in its own PR so it can be landed quickly. I [mentioned this](https://github.com/NatronGitHub/Natron/pull/1039#discussion_r2060321652) in the previous incarnation of these changes.